### PR TITLE
Add coverage script

### PR DESCRIPTION
**Pull request by jimf**
Dunno if there's interest in this, but I went ahead and added istanbul integration for getting a more concrete idea of existing test coverage via `npm run coverage`, which gives nearly identical output as the existing `npm test`, but with a console summary of overall coverage, as well as some reports to `coverage/`.

Feedback welcome!

image

### CODE REVIEW

**Comment by buzzdecafe** [comment]
:cow2:

**Comment by CrossEye** [comment]
No objection, and no strong desire for this either. :+1: :-1:

File: package.json [hunk 1]
```diff
@@ -57,6 +60,7 @@
     "escodegen": "1.4.x",
     "eslint": "^1.10.1",
     "handlebars": "3.0.x",
+    "istanbul": "^0.4.2",
```
Comment (davidchambers) [inline package.json]: `"0.4.x"`, please.

**Comment by buzzdecafe** [comment]
apart from needing the commits squashed, any reason not to merge this?

**Comment by davidchambers** [comment]
I'm in favour of this change. I'd like to require 100% branch coverage; this is a baby step in that direction.

**Comment by buzzdecafe** [comment]
agreed. @jimf please squash to one commit and i'll merge

**Comment by jimf** [comment]
@davidchambers @buzzdecafe :zap: Squashed

**Comment by davidchambers** [comment]
:deciduous_tree:

**Comment by buzzdecafe** [comment]
thanks

**Pull request merged**